### ISSUE TITLE
Cannot install func-adl-servicex with Python 3.8 on Windows

### ISSUE THREAD

**Issue by AndrewEckart** [iris-hep/func_adl_servicex#12]
![image](https://user-images.githubusercontent.com/5485577/96625843-51392b80-12d4-11eb-9f4b-42c3330e4d9e.png)

**Comment by AndrewEckart** [comment]
@oshadura experienced this issue as well. The error message is slightly different (and more illuminating) on linux, but I think it's the same problem.

```
jovyan@jupyter-oksana-2eshadura-40cern-2ech:~$ pip install func-adl-servicex
Collecting func-adl-servicex
  Using cached func_adl_servicex-1.0a5-py3-none-any.whl (5.3 kB)
Requirement already satisfied: servicex<3.0a1,>=2.1b1 in /opt/conda/lib/python3.8/site-packages (from func-adl-servicex) (2.1b4)
Requirement already satisfied: qastle==0.8 in ./.local/lib/python3.8/site-packages (from func-adl-servicex) (0.8)
Requirement already satisfied: func-adl>=2.0a1 in /opt/conda/lib/python3.8/site-packages/func_adl-2.0b1-py3.8.egg (from func-adl-servicex) (2.0b1)
ERROR: Package 'func-adl' requires a different Python: 3.8.5 not in '>=3.6, <3.8'
jovyan@jupyter-oksana-2eshadura-40cern-2ech:~$ 
```

**Comment by gordonwatts** [comment]
This is by design, I'm afraid. This will not be fixed until we move everything up to python 3.8.

**Comment by gordonwatts** [comment]
See https://github.com/iris-hep/func_adl/issues/15 - that is the issue to track this on - that makes sure that we are working on the user side (the transformer side is disconnected).
